src/__tests__/useTween.test.ts

```diff
@@ -0,0 +1,56 @@
+import { renderHook } from '@testing-library/react-hooks';
```

Comment:
Similar to useSpring tests: I'm mocking useRaf hook and easing fn and checking everything is init properly, as it's actually what this hook is doing (though again this is testing implementation rather than behavior but I think it's fine for this one).